pkg/provider-local/webhook/controlplane/templates/scripts/configure-containerd.tpl.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Well, it's quite unfortunate that we cannot use https://github.com/gardener/gardener/blob/master/docs/usage/custom-containerd-config.md even ourselves anymore, or can we? If one has to tamper with the default `config.toml` file to get this to work, then this whole mechanism of importing and merging additional `.toml` files is useless and should be dropped.

Reply:
Granted that someone is aware of the fact that sections are merged and not individual keys, it could still be used to overwrite certain configurations without touching Gardener's config.toml.
It's problematic for the CRI section as Gardener has a non-default configuration for this section.

I can't  judge whether Gardener should drop this `import` functionality based  on that described behavior.

Reply:
From my POV, the cleanest solution is an API contract offered by Gardener, without directly touching the containerd config file.

Reply:
/cc @timebertt - opinions?

Reply:
I agree with @rfranzke. It is very unfortunate, that containerd doesn't properly merge the config files. I'm not sure we were aware about this when introducing the imports functionality.

That being said, I would keep the imports functionality to spare us the effort of deprecation and breaking removal. As said, users might still find corner cases for making use of it (very few though).
However, the documentation should be adapted and clearly warn users about the trap.

I think the approach @danielfoehrKn took in this PR makes sense for provider-local and should be good enough.
Considering the options containerd offers, I don't see a better way for how extensions can contribute to the containerd config.